### DISCUSSION TITLE
the audio is played from the start when I press play after being paused

### DISCUSSION THREAD

**Discussion by mohamadabdelhady** [goldfire/howler.js#1571] [Q&A]
I used howlerjs to build an audio player on my website, the problem is whenever I pause the audio then play it start from the beginning of the audio, this happen only on google chrome it works fin in fire fox, does somebody have a solution.

**Comment by nsw42** [comment]
I've been encountering what seems like the same issue. (Vivaldi browser, rather than Chrome, but they're both Chromium-based.)  This with v2.2.3.

I found that removing the assignment to `node.currentTime` on line 900 (as at commit 2016300), inside `playHtml5()`, worked for me.

My app/site doesn't support seeking, just pause/resume, so this is a sufficient fix for me, but it seems fragile and I've therefore not submitted it as a PR.
